### PR TITLE
Update the description of what's in DK

### PR DESCRIPTION
**Pull request by tas50**
Remove Berkshelf. Remove some funky things that didn't make any sense. Add our test apps.

### CODE REVIEW

File: chef_master/source/install_dk.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ Install the Chef DK
 =====================================================
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/install_dk.rst>`__
 
-The Chef DK installer is used to set up the Chef development kit on a workstation, including the chef-client itself, an embedded version of Ruby, RubyGems, OpenSSL, key-value stores, parsers, libraries, command line utilities, and community tools such as Kitchen, Berkshelf, and ChefSpec. The Chef installer puts everything into a unique directory (``/opt/chefdk/`` on macOS / Linux and ``C:\opscode\chefdk\`` on Windows) so that these components will not interfere with other applications that may be running on the target machine.
+Use the Chef DK installer to set up the Chef development kit on a workstation. Chef DK includes the chef-client, an embedded version of Ruby, RubyGems, and OpenSSL, as well as our tools: Test Kitchen, Cookstyle, Foodcritic and ChefSpec. The Chef installer puts everything into a unique directory (``/opt/chefdk/`` on macOS / Linux and ``C:\opscode\chefdk\`` on Windows) so that these components will not interfere with other applications that may be running on the target machine.
```
Comment (mjingle) [inline chef_master/source/install_dk.rst:6]: @tas50 Are there other tools to mention at the end of the second sentence? And how does the first two sentences look to you?

Comment (tas50) [reply chef_master/source/install_dk.rst:6]: This looks good

**Comment by tas50** [comment]
LGTM

**Review by mjingle — APPROVED** [review]
tenor-102785550

**Pull request merged**